### PR TITLE
Prepare for k8s release 1.19 (Aug 25)

### PR DESCRIPTION
**Pull request by dholbach**
This is #152, #139, #127 in disguise.

Maybe this time let it simmer a bit longer until the k8s release is there?

Testing is still welcome!

### CODE REVIEW

**Comment by dholbach** [comment]
Let's test this against 1.19.0, 1.18.8, 1.17.11.

**Comment by dholbach** [comment]
You can use `weaveworks/kured:prep-for-k8s-1.19-84c97e6`.

**Comment by dholbach** [comment]
:heavy_check_mark: 1.19.0 (minikube).

File: cmd/kured/main.go [hunk 1]
```diff
@@ -257,7 +258,7 @@ func uncordon(nodeID string) {
 }
 
 func commandReboot(nodeID string) {
-	log.Infof("Commanding reboot")
+	log.Infof("Commanding reboot for node: %s", nodeID)
```
Comment (dholbach) [inline cmd/kured/main.go:261]: This was a mistake on my part. Unfortunately I reverted #134 in #135 again. :facepalm:

Comment (dholbach) [reply cmd/kured/main.go:261]: It'll be good to put it back in.

**Comment by dholbach** [comment]
`docker.io/weaveworks/kured:prep-for-k8s-1.19-3ebc224`

**Comment by dholbach** [comment]
:heavy_check_mark: `1.18.8 (minikube)`.

**Comment by dholbach** [comment]
:heavy_check_mark: `1.17.11 (minikube)`.

**Comment by dholbach** [comment]
I'm having trouble re-testing 1.19.0 (weird - it worked earlier(?)), because of https://github.com/kubernetes/kubeadm/issues/2078#issuecomment-682420921.

**Comment by evrardjp** [comment]
Worked for me on 1.18.x
Testing now 1.19.0

**Comment by evrardjp** [comment]
Worked for me on 1.19.0 in kind.

**Review by evrardjp — APPROVED** [review]
LGTM in terms of features.

However, because the chart might have changed on a comment, I expect that it will require a new patch version to pass the CI. :)

**Comment by dholbach** [comment]
Thanks for the review. And yes, we'll need another "bump versions PR" to actually make this releasable. In the meantime, I'd like to get this merged, so our Dev environment can pick it up and I can also play with it there.

**Pull request merged**